app/assets/javascripts/ui_models/application_group.ts

```diff
@@ -125,7 +124,8 @@ export class ApplicationGroup {
     }
 
     return () => {
-      pull(this.changeObservers, callback)
+      const indexOfObserver = this.changeObservers.indexOf(callback);
+      this.changeObservers.splice(indexOfObserver, 1);
```

Comment:
Should use `removeFromArray`. We use this 14 times already in the web code. No need to use splice directly.